install.sh

```diff
@@ -85,13 +90,59 @@ pacman --noconfirm -S \
 	grub \
 	steam
 
-# install nvidia graphics if needed
-if lspci | grep -E -i '(vga|3d)' | grep -i nvidia > /dev/null; then
-	pacman --noconfirm -S nvidia nvidia-utils lib32-nvidia-utils
-fi
-
 systemctl enable NetworkManager lightdm bluetooth
 
+# gpu detection/driver installation
+elif lspci -nnk | grep -i vga -A3 | grep 'Kernel modules: radeon, amdgpu' > /dev/null; then
+	echo "AMD GPU with radeon kernel detected, switching kernel module and installing drivers..."
+	echo "blacklist radeon" > /etc/modprobe.d/blacklist.conf
+	echo "options amdgpu si_support=1" > /etc/modprobe.d/amdgpu.conf
+	echo "options amdgpu cik_support=1" >> /etc/modprobe.d/amdgpu.conf
+	echo "options radeon si_support=0" > /etc/modprobe.d/radeon.conf
+	echo "options radeon cik_support0" >> /etc/modprobe.d/radeon.conf
+	pacman --noconfirm -S \
+		vulkan-radeon mesa \
+		vulkan-icd-loader \
+		xf86-video-amdgpu \
+		libva-mesa-driver \
+		mesa-vdpau \
+		lib32-libva-mesa-driver \
+		lib32-mesa-vdpau \
+		lib32-mesa \
+		lib32-vulkan-radeon
+
+elif lspci -nnk | grep -i vga -A3 | grep 'Kernel modules: radeon' > /dev/null; then
```

Comment:
Same comment: will this ever be true when installing from the Arch linux installer which is command line only?

Reply:
@tokyoraven02 wrote this feature with the Arch Linux wiki in mind.


This is a reasonable concern, albeit I don't believe that the fact that this script is supposed to be executed within the Arch Linux installer is going to make a difference, since we're talking about kernel modules over here.

We are in need of testers.